classes/PodsInit.php

```diff
@@ -362,6 +353,18 @@ public function register_assets() {
 		wp_register_style( 'pods-wizard', PODS_URL . 'ui/styles/dist/pods-wizard.css', array(), PODS_VERSION );
 		wp_register_style( 'pods-form', PODS_URL . 'ui/styles/dist/pods-form.css', array(), PODS_VERSION );
 
+		// Deal with specifics on admin pages
```

Comment:
What about `$register_handlebars` usage here?

Reply:
Oh, yeah, that was rather negligent of me